make/hotspot/lib/JvmFeatures.gmk

```diff
@@ -123,6 +123,8 @@ ifneq ($(call check-jvm-feature, cds), true)
   JVM_EXCLUDE_FILES += \
       classLoaderDataShared.cpp \
       classLoaderExt.cpp \
+      precompiler.cpp \
```

Comment:
Oh nice! I didn't know about this. Yesterday I was sprinkling INCLUDE_CDS in SCCache.cpp because it was getting compiled in the minimal config.

Reply:
Yup. This is a nuclear option in the build system! For your other PR, you might need to stub out more things with `NOT_CDS*` in `SCCache.hpp` to make things work.